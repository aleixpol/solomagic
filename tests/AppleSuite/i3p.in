\ref something0
\mt somethingsomething indi somethingsomething

\ref something1
\mt somethingsomething india somethingsomething

