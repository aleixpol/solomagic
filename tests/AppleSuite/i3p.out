\ref something0
\mt somethingsomething indi somethingsomething
\ma somethingsomething ingri somethingsomething

\ref something1
\mt somethingsomething india somethingsomething
\ma somethingsomething india somethingsomething
